SwiftUIHW/SwiftUIHW/AccountDetailView/AccountDetailViewModel.swift

```diff
@@ -0,0 +1,142 @@
+// AccountDetailViewModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Вью модель экрана с картой
+final class AccountDetailViewModel: ObservableObject {
+    // MARK: - Public properties
+
+    @Published var cardNumber = Constants.defaultCardNumber
```

Comment:
```suggestion
    @Published var cardNumber = 
```
так вот не надо см важное